packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -667,7 +675,7 @@ export interface CallAdapterSubscribers {
   /**
    * Subscribe function for 'captionsPropertyChanged' event.
    */
-  on(event: 'captionsPropertyChanged', listener: PropertyChangedEvent): void;
+  on(event: 'captionsPropertyChanged', listener: CaptionsPropertyChangedEventListener): void;
```

Comment:
Maybe just name it like PropertyChangedEventListener?

Or directly do
```suggestion
  on(event: 'captionsPropertyChanged', listener: () => void): void;
```?

Hmmm, () => void is event less characters than PropertyChangedEventListener

Reply:
Or... we should directly use the same name isCaptionsActiveChanged (is it the right name coming from sdk?)

Then do 

IsCaptionsActiveChangedEventListener = (event: {isActive: boolean} ) => void

 an event listener + an event makes more sense to get new type

Reply:
just checked what we bubble up from caption feature, the event is isCaptionsActiveChanged

Let's use the same name here and add isActive as event payload 

And don't forget to change AzureCommunicationCallAdapter to update the implementation  la bit